src/app/pages/countries/countries.component.html

```diff
@@ -39,49 +36,58 @@
       <div class="align-middle inline-block min-w-full shadow overflow-hidden sm:rounded-lg border-b border-gray-200">
         <table class="min-w-full">
           <thead>
-          <tr>
-            <th class="px-6 py-3 border-b border-gray-200 bg-white text-left text-xs leading-4 font-medium text-gray-600 uppercase tracking-wider">
-              Pays
-            </th>
-            <th class="px-6 py-3 border-b border-gray-200 bg-white text-left text-xs leading-4 font-medium text-gray-600 uppercase tracking-wider">
-              Confirmés
-            </th>
-            <th class="px-6 py-3 border-b border-gray-200 bg-white text-left text-xs leading-4 font-medium text-gray-600 uppercase tracking-wider">
-              guérisons
-            </th>
-            <th class="px-6 py-3 border-b border-gray-200 bg-white text-left text-xs leading-4 font-medium text-gray-600 uppercase tracking-wider">
-              Morts
-            </th>
-          </tr>
+            <tr>
+              <th
+                class="px-6 py-3 border-b border-gray-200 bg-white text-left text-xs leading-4 font-medium text-gray-600 uppercase tracking-wider">
+                Pays
+              </th>
+              <th
+                class="px-6 py-3 border-b border-gray-200 bg-white text-left text-xs leading-4 font-medium text-gray-600 uppercase tracking-wider">
+                Confirmés
+              </th>
+              <th
+                class="px-6 py-3 border-b border-gray-200 bg-white text-left text-xs leading-4 font-medium text-gray-600 uppercase tracking-wider">
+                guérisons
+              </th>
+              <th
+                class="px-6 py-3 border-b border-gray-200 bg-white text-left text-xs leading-4 font-medium text-gray-600 uppercase tracking-wider">
+                Morts
+              </th>
+            </tr>
           </thead>
           <tbody class="bg-white">
-          <tr *ngFor="let country of data | filter : searchText : sortBy" routerLink="/countries/{{ country.url }}" [state]="{data: {country: country}}">
-            <td class="px-6 py-4 whitespace-no-wrap border-b border-gray-200" >
+            <tr
+              *ngFor="let country of data | filter : searchText | paginate: { itemsPerPage: 10, currentPage: currentPage }"
+              routerLink="/countries/{{ country.url }}" [state]="{data: {country: country}}">
```

Comment:
Can you also add the current page in the url so that when you refresh you still get the same view such as `/countries/{{country.url}}?page=currentPage`?

Reply:
sure, let me take a look on that.